base/bitarray.jl

```diff
@@ -1276,8 +1274,17 @@ function mod(x::Number, B::BitArray)
     y = mod(x, true)
     return fill(y, size(B))
 end
+function broadcast(::typeof(div), B::BitArray, x::Number)
+    T = promote_op(div, Bool, typeof(x))
+    T === Any && return [div(b, x) for b in B]
+    F = Array{T}(size(B))
+    for i = 1:length(F)
+        F[i] = div(B[i], x)
+    end
+    return F
+end
```

Comment:
Did #17623 nix a former version of this method? If so, I will remove this method.